lib/interfaces/src/VCFInterface.cpp

```diff
@@ -0,0 +1,25 @@
+#include "VCFInterface.h"
+#include "hytech.h"
+
+void VCFInterface::receive_pedals_message(const CAN_message_t& msg, unsigned long curr_millis)
+{
+    VCR_PEDALS_t vcr_pedals_msg;
+    Unpack_VCR_PEDALS_hytech(&vcr_pedals_msg, &msg.buf[0], msg.len);
+    _curr_data.pedals_data.recvd_data.implausibility_has_exceeded_max_duration = vcr_pedals_msg.implaus_exceeded_max_duration;
```

Comment:
Why have there be another intermediate step of VCFInterface's _curr_data? Why can we not pass in a reference to the global interface_data and allow this function to report this data to a global state?